plugins/influxdbjson/influxdb_json.go

```diff
@@ -0,0 +1,161 @@
+package influxdbjson
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"strings"
+	"sync"
+
+	"github.com/influxdb/telegraf/plugins"
+)
+
+type InfluxDBJSON struct {
+	Name string
+	URLs []string `toml:"urls"`
+}
+
+func (*InfluxDBJSON) Description() string {
+	return "Read InfluxDB-formatted JSON metrics from one or more HTTP endpoints"
+}
+
+func (*InfluxDBJSON) SampleConfig() string {
+	return `
+	# Reads InfluxDB-formatted JSON from given URLs. For example,
```

Comment:
Most of this seems like it belongs more in the README, telegraf config file would be tens of thousands of lines if we had in/out examples for each plugin